app/src/main/java/com/example/neighborly/BuildingModel.java

```diff
@@ -74,6 +70,19 @@ public void setRequestList(List<RequestModel> requestList) {
         this.requestList = requestList;
     }
 
+    public void addRequestToList(RequestModel requestModel) {
+        requestList.add(requestModel);
+    }
+
+    public RequestModel getLastRequest(){
```

Comment:
Maybe change to getLastRealRequest or add a comment stating this is due to null refs in firebase